.github/workflows/smoketest.yml

```
@@ -38,11 +38,11 @@ jobs:
             ${{ runner.os }}-yarn-
 
       - name: Install dependencies on cache miss
-        run: yarn install --frozen-lockfile
+        run: yarn install
```

Immutable is the replacement for frozen lockfile.

```suggestion
        run: yarn install --immutable
```

---

Yeah I did that. The build had a checksum error I couldn't solve.

---

See 25b28fbea5e519f1e7256da80805310a11ccfd5c

---

Likely needs a lockfile generated on Linux.

---

VSCodespaces to the rescue then!